README.md

```diff
@@ -50,6 +50,9 @@ Then run `build_tfm.py` with the PSA API config.
 python3 build_tfm.py -m CY8CKIT_064S2_4343W -t GNUARM -c ConfigPsaApiTestIPC.cmake -s CRYPTO
 ```
 
+Note: Make sure the TF-M Regression Test suite has PASSED on the board before
+running any PSA Compliance Test suite to avoid unpredictable behavior.
```

Comment:
Can't the script verify this automatically?

Reply:
Yes, it can only for `ARM_MUSCA_B1` ... but users can run these `build_tfm.py` and `build_psa_compliance.py` scripts on a standalone basis for other targets during development/porting stages.

Reply:
OK, this is general documentation not `test_psa_target.py` documentation